src/plugins/kibana/public/visualize/editor/draggable_container.js

```diff
@@ -0,0 +1,85 @@
+import _ from 'lodash';
+import $ from 'jquery';
+import dragula from 'dragula';
+import uiModules from 'ui/modules';
+
+uiModules
+.get('app/visualize')
+.directive('draggableContainer', function () {
+
+  return {
+    restrict: 'A',
+    scope: true,
+    controllerAs: 'draggableContainerCtrl',
+    controller ($scope, $attrs, $parse) {
+      this.getList = $parse($attrs.draggableContainer);
```

Comment:
Generally it's easier to do this type of thing with a `&` binding on an isolated scope, is that an option here?